features/steps/common_http.py

```diff
@@ -160,7 +160,9 @@ def set_rest_api_prefix(context, prefix):
 @when("I access endpoint {endpoint} using HTTP GET method")
 def access_rest_api_endpoint_get(context, endpoint):
     """Send GET HTTP request to tested service."""
-    url = f"http://{context.hostname}:{context.port}/{context.api_prefix}/{endpoint}"
+    base = f"http://{context.hostname}:{context.port}"
+    path = f"{context.api_prefix}/{endpoint}".replace("//", "/")
```

Comment:
replaceall?

Reply:
WDYM? I'm replacing all the double slash in the subdirectory, not the ones in `http://`

Reply:
the purpose is to avoid things like `http://localhost:8000//api/v1//rendered_reports`

Reply:
sure, but those `//` might be used multiple times (and not - I'm not talking about `base` part)